File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/ListViews_ViewMyCheckboxesRow.cpp

#include "ListViews_ViewMyCheckboxesRow.h"
#include "ListViews_ViewMyCheckboxesRow.h"
#include <string>

namespace widgetassertions
{
  bool ListViews_ViewMyCheckboxesRow::getIsHeaderCheckBoxChecked( ) 
  {
    return this->isHeaderCheckBoxChecked;
  }
  std::string ListViews_ViewMyCheckboxesRow::getHeaderCheckBoxText( ) 
  {
    return this->HeaderCheckBoxText;
  }
  int ListViews_ViewMyCheckboxesRow::getRowIndex( ) 
  {
    return this->RowIndex;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/ListViews_ViewMyCheckboxesRow.cs
/// <filename>
///     ListViews_ViewMyCheckboxesRow.cs
/// </filename>


namespace widgetassertions
{
    internal class ListViews_ViewMyCheckboxesRow
  {
        private bool isHeaderCheckBoxChecked;
     
        public     virtual bool getIsHeaderCheckBoxChecked()
    {
      return this.isHeaderCheckBoxChecked;
    }
     
        private string HeaderCheckBoxText;
     
        public     virtual string getHeaderCheckBoxText()
    {
      return this.HeaderCheckBoxText;
    }
     
        private int RowIndex;
     
        public     virtual int getRowIndex()
    {
      return this.RowIndex;
    }
  }
}

